packages/app-accounts/src/Editor.js

```diff
@@ -67,6 +67,7 @@ class Editor extends React.PureComponent<Props, State> {
 
     return (
       <Button.Group>
+        <Button.Group.Divider style={{ padding: '0em' }} />
```

Comment:
Is it possible that we can move this to the base component- feels a bit wrong having to do this everywhere?